client/src/pages/learn/coding-interview-prep/algorithms/index.md

```diff
@@ -5,16 +5,16 @@ superBlock: Coding Interview Prep
 ---
 ## Introduction to the Coding Interview Prep Algorithms
 
-These challenges are meant to teach and challenge you with common algorithms that you will encounter in real life. Hopefully, these challenges will be a great learning experience and your programming and logical skills will improve.
+These challenges are meant to teach and challenge you with common Algorithms that you will encounter in real life. Hopefully, these challenges will be a great learning experience and your programming and logical skills will improve.
 
-In mathematics and computer science, an algorithm (/ˈælɡərɪðəm/ ) is an unambiguous specification of how to solve a class of problems. Algorithms can perform calculation, data processing and automated reasoning tasks.
-In most of the coding interviews, foundation mostly means clear understanding of basic data structures and algorithms.
+In mathematics and computer science, an Algorithm (/ˈælɡərɪðəm/ ) is an unambiguous specification of how to solve a class of problems. Algorithms can perform calculation, data processing and automated reasoning tasks.
+In most of the coding interviews, foundation mostly means a clear understanding of basic 3data structures and Algorithms.
 
 This course presents you with the most asked questions during the interviews to prepare your basic knowledge of Algorithms.
 It will provide a clear and concise understanding and implementations of different Algorithms.
 
-# Prerequisite: All the Algorithmic questions would require the solutions in Javascript. So, it's a good idea to complete the javascript course first.
+# Prerequisite: All the Algorithmic questions would require the solutions in Javascript. So, it's a good idea to complete the Javascript course first.
 
-It's a very good practise to use a whiteboard or a notepad to implement and practise the interview questions. Since most of the coding interviews use either a Whiteboard or a Notepad for writing the codes. So forget your nice IDEs and debuggers and start getting your hands dirty.
+It's a very good practice to use a Whiteboard or a Notepad to implement and practice the interview questions, since most coding interviews use either a Whiteboard or a Notepad for writing the code. So forget your nice IDEs and debuggers and start getting your hands dirty.
```

Comment:
Spelling correction -- practise to practice. Capitalization of `Javascript` also good.

Reply:
I would argue it sounds more correct to say `It's very good practice` i.e. removing the a